src/frontend/Komponenter/Behandling/Totrinnskontroll/FatterVedtak.tsx

```diff
@@ -51,16 +56,19 @@ enum Totrinnsresultat {
 }
 
 const FatterVedtak: React.FC<{
-    behandlingId: string;
+    behandling: Behandling;
     settVisGodkjentModal: (vis: boolean) => void;
-}> = ({ behandlingId, settVisGodkjentModal }) => {
+}> = ({ behandling, settVisGodkjentModal }) => {
     const [godkjent, settGodkjent] = useState<Totrinnsresultat>(Totrinnsresultat.IKKE_VALGT);
     const [årsakerUnderkjent, settÅrsakerUnderkjent] = useState<ÅrsakUnderkjent[]>([]);
     const [begrunnelse, settBegrunnelse] = useState<string>();
     const [feil, settFeil] = useState<string>();
     const [laster, settLaster] = useState<boolean>(false);
+    const [simuleringStatusFeil, settSimuleringStatusFeil] = useState<boolean>(false);
```

Comment:
Her ville jeg gitt samme navn som i backend, for å lettere vise hva det er vi sjekker:
```suggestion
    const [erSimuleringsresultatEndret, settErSimuleringsresultatEndret] = useState<boolean>(false);
```